app/views/alterations_amends.scala.html

```diff
@@ -28,55 +29,52 @@
     @reference(requestObject.getPageTitle)
     <p class="font-xsmall"><a href="@ersUtil.getPageBackLink(schemeId, ersUtil.PAGE_ALT_AMENDS)" class="link-back">@Html(messages("ers.back"))</a></p>
 
-    @header(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "question"))
     @if(flash.get("alt-amends-not-selected-error").isDefined) {
-    <div id="errors" class="validation-summary" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
-        <h1 class="heading-medium error-summary-heading" id="error-summary-heading">
-            @Html(messages("validation.summary.heading"))
-        </h1>
-        <ul class="validation-summary-message">
-            <li class=validation-summary-message"><a href="#error-summary" data-focuses="error-summary">@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "err.message"))</a></li>
-        </ul>
-    </div>
+        <div class="flash error-summary error-summary--show " id="error-summary-display" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@messages("validation.summary.heading")</h2>
+            <ul class="js-error-summary-messages">
+                <li>
+                    <a href="#error-summary" data-focuses="error-summary">@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "err.message"))</a>
+                </li>
+            </ul>
+        </div>
     }
 
-    @uk.gov.hmrc.play.views.html.helpers.form(action = routes.AltAmendsController.altAmendsSelected()) {
-        <div class="form-group @if(flash.get("alt-amends-not-selected-error").isDefined) {error}">
+    @helpers.form(action = routes.AltAmendsController.altAmendsSelected()) {
+        <div class="form-group @if(flash.get("alt-amends-not-selected-error").isDefined) {form-field--error}">
             <fieldset>
                 <legend>
+                    @header(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "question"))
                     <span>@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "instruction"))</span>
                 </legend>
                 @if(flash.get("alt-amends-not-selected-error").isDefined) {
-                <div aria-labelledby="error-heading" class="error-message" id="error-summary" role="alert" tabindex="-1">
```

Comment:
Should the id have been removed here? The <a> above was linking to it before

Reply:
I'll have a look, might have accidently removed it

Reply:
added it back in